pkg/controller/migplan/migplan_controller.go

```diff
@@ -344,3 +350,37 @@ func (r ReconcileMigPlan) ensureClosed(plan *migapi.MigPlan) error {
 
 	return nil
 }
+
+// Get whether the plan is `suspended`.
+// A plan is considered`suspended` when a migration is running or the final migration has
+// completed successfully. While suspended, reconcile is limited to basic validation
+// and PV discovery and ensuring resources is not performed.
+func (r *ReconcileMigPlan) planSuspended(plan *migapi.MigPlan) error {
```

Comment:
This function really does more than 'get' whether a plan is suspended, it also sets the suspended condition depending on migrations associated with the plan. The wording in the comment confused me initially.

Reply:
It originally determined whether the plan was suspended and returned a boolean.  Then, updated to set the condition.  I s/Get/Determine/.  Does that make it clear enough?

Reply:
I think 'determine' makes it clearer than 'get' for sure :+1: